### PR TITLE
Spanish Translations

### PR DESCRIPTION
**Pull request by egzonqj**
**Please describe your improvement.**
Commiting a bunch of translations. I might revise later some past translations that I believe could be better by applying the proper context, or just applying consensus on similar texts.

### CODE REVIEW

File: PKHeX.WinForms/Resources/text/lang_es.txt [hunk 1]
```diff
@@ -37,7 +37,7 @@ SAV_PokedexORAS=Editor de Pokédex (ORAS)
 SAV_PokedexSM=Editor de Pokédex 
 SAV_PokedexSWSH=Pokédex Editor
```
Comment (NotTsunami) [inline PKHeX.WinForms/Resources/text/lang_es.txt]: `Editor de Pokédex (SWSH)`

File: PKHeX.WinForms/Resources/text/lang_es.txt [hunk 2]
```diff
@@ -404,7 +404,7 @@ SAV_Database.Menu_Tools=Herramientas
 SAV_Database.Tab_Advanced=Advanced
```
Comment (NotTsunami) [inline PKHeX.WinForms/Resources/text/lang_es.txt]: All instances of Advanced can be `Avanzado`

File: PKHeX.WinForms/Resources/text/lang_es.txt [hunk 3]
```diff
@@ -893,32 +893,32 @@ SAV_PokedexSM.GB_Encountered=Visto
...
-SAV_PokedexSWSH.B_Save=Save
-SAV_PokedexSWSH.CHK_Caught=Owned
-SAV_PokedexSWSH.CHK_G=Gigantamax
+SAV_PokedexSWSH.B_Cancel=Cancelar
+SAV_PokedexSWSH.B_GiveAll=Marcar todos
+SAV_PokedexSWSH.B_Modify=Modificar...
+SAV_PokedexSWSH.B_Save=Gaurdar
+SAV_PokedexSWSH.CHK_Caught=Capturado
+SAV_PokedexSWSH.CHK_G=Gigamax
 SAV_PokedexSWSH.CHK_Gigantamaxed=Gigantamaxed
```
Comment (NotTsunami) [inline PKHeX.WinForms/Resources/text/lang_es.txt]: I _believe_ this should be `Gigamaxizó`, considering the official translation for the verb is gigamaxizar.

Comment (egzonqj) [reply PKHeX.WinForms/Resources/text/lang_es.txt]: I believe too, I couldn't check back when I posted this several hours ago, thanks either way!

File: PKHeX.WinForms/Resources/text/lang_es.txt (hunk 1 above)
Comment (egzonqj) [reply PKHeX.WinForms/Resources/text/lang_es.txt]: I might want to check these, since I don't really think adding the games are necessary there?

**Comment by egzonqj** [comment]
I believe everything's better now, applied better context to most of the texts. I guess this can be heavily improved but I think it's OK as for now.

**Comment by kwsch** [comment]
As always, feel free to submit another PR if more updates are desired :)

Thanks!

**Comment by egzonqj** [comment]
Sure thing!

**Pull request merged**